home_assistant_glow.yaml

```diff
@@ -137,3 +144,9 @@ sensor:
         # multiply value = 1 / imp value
         # - multiply: 0.001
         - lambda: return x * (1.0 / ${pulse_rate});
+
+        # Update the sensor once per 0.1 kWh consumed, or every 5th minute,
+        # whichever happens sooner. See https://github.com/klaasnicolaas/home-assistant-glowReadme.md#reduce-the-amount-of-data-the-sensors-produce
+        # for more information.
```

Comment:
Would this approach be a desirable solution? It seems to me to be quite aggressive in data filtering and how reliable is your data if you are going to use this 🤔

Reply:
In what way do you think it wouldn't be reliable? You won't have all the raw data in Home Assistant of course, but that shouldn't matter in practice. Restarting the ESPHome device may mean that up to < 0.1 kWh of data is lost too.

But you have a point, the Home Assistant energy dashboard uses kWh with 2 decimals, so it could probably make sense to set the (commented out) default here to report once per every 0.01 kWh instead. I'll go ahead and make that change.

Reply:
My mistake I looked wrong and had the idea that this also means that the power entity gets updates less often, once every 5 minutes (while you would like to have that up to date). But that's not the case at all, so this seems fine to me.

Reply:
👍 

Rebased this PR on top of latest main.